docs/index.md

```diff
@@ -1,4 +1,164 @@
 HCA DCP Query Service
 =====================
 
-[User Stories](USER_STORIES.md)
+# Introduction
+
+The HCA DCP Query Service provides an interface for scientists and developers to query metadata associated with
+experimental and analysis data stored in the [Human Cell Atlas](https://staging.data.humancellatlas.org/)
+[Data Coordination Platform](https://www.humancellatlas.org/data-sharing) (DCP). Metadata from the
+[DCP Data Store](https://github.com/HumanCellAtlas/data-store) are indexed and stored in an
+[AWS Aurora](https://aws.amazon.com/rds/aurora/) [PostgreSQL](https://www.postgresql.org/) database.
+
+# Query via the API
+Queries to the database can be sent over HTTP through the Query Service API
+[Swagger Documentation](https://query.data.humancellatlas.org/v1/ui/#/)
+or via the [Query Builder](https://query.data.humancellatlas.org/)
+
+## Executing Queries
+To execute a query via the swagger interface 
```

Comment:
Please add a colon or comma at the end of this line.

Reply:
👍